io/pci/dlpar.py

```diff
@@ -67,6 +67,12 @@ def setUp(self):
         if not self.server:
             self.cancel("Managed System not got")
         self.lpar_2 = self.params.get("lpar_2", default=None)
+        self.pci_device = self.params.get("pci_device", default=None).split(' ')
+
+    def do_test_setup(self, pci_device):
```

Comment:
what does this funciton do ?   instead of do_test_setup() may be some meaning full name i.e validate or reset or recover ?

Reply:
@abdhaleegit the function doesn't validate any thing. The function does all the prerequisite setup. self.loc_code, self.drc_index, self.lpar_id, self.phb All these parameters are used in dlpar add/remove/move commands which are obtained in this function. Hence i made it as "do_test_setup"
Let me know what can be given for setting up details instead of "do_test_setup"
Or how about "obtain_values" ?

Reply:
Modified the function name to "set_adapter_details"